lesson1/src/main/java/ru/geekbrains/lesson1/MainApp.java

```diff
@@ -0,0 +1,79 @@
+package ru.geekbrains.lesson1;
+
+public class MainApp {
+    public static void main(String[] args) {
+        //Задание 2. Инициализируем переменные всех типов данных из урока
+        byte byteValue = 127;
+        short shortValue = 32767;
+        int intValue = 2147483647;
+        long longValue = 9223372036854775807L;
+        float floatValue = 2147483647.0f;
+        double doubleValue = 9223372036854775807.000;
+        boolean booleanValue = true;
+        char charValue = 'a';
+        String stringValue = "test";
+
+        //Тестируем работу методов
+        System.out.println("Задание 3. Проверяем метод somethingCalculate");
+        System.out.println(somethingCalculate(5,4,3,2));
+
+        System.out.println("Задание 4. Проверяем метод returnBoolean");
+        System.out.println(returnBoolean(5, 5));
+        System.out.println(returnBoolean(20, 20));
+
+        System.out.println("Задание 5. Проверяем метод printSomething");
+        printSomething(5);
+        printSomething(-5);
+
+        System.out.println("Задание 6.Проверяем метод resultBoolean");
+        System.out.println(resultBoolean(5));
+        System.out.println(resultBoolean(-5));
+
+        System.out.println("Задание 7. Проверяем метод helloName");
+        helloName("Иван");
+
+        System.out.println("Задание 8. Проверяем метод whatYear");
+        whatYear(2000);
+    }
+    //Метод возвращает результат вычисления в формате float
+    public static float somethingCalculate (float a, float b, float c, float d) {
+
+        return a * (b + (c / d));
+    }
+
+    //Метод возвращает результат проверки суммы чисел на вхождение в диапазон
+    public static boolean returnBoolean (int a, int b) {
+
+        return (a + b) >= 10 && (a + b) <= 20;
+    }
+
+    //Метод проверяет положительное число или отрицательное и выводит результат в консоль
+    public static void printSomething (int a) {
+        if (a >= 0) {
+            System.out.println("Положительное число");
+        }
+        if (a < 0) {
```

Comment:
```suggestion
        else {
```
Поскольку это единственный альтернативный вариант для первого условия